examples/v2/cloudbuild/jinja/cloudbuild.jinja

```diff
@@ -0,0 +1,15 @@
+resources:
```

Comment:
This one fails for me:

kpedelty@curious-scarab-126723:~$ gcloud deployment-manager deployments create my-build --config /tmp/cloudbuild.yaml
The fingerprint of the deployment is 6Ed2qzRM5Qvxk1Awpx1M0A==
Waiting for create [operation-1510604002213-55de2e6f2bc89-def837fb-5cba0d33]...failed.
ERROR: (gcloud.deployment-manager.deployments.create) Error in Operation [operation-1510604002213-55de2e6f2bc89-def837fb-5cba0d33]: errors:
- code: RESOURCE_ERROR
  location: /deployments/my-build/resources/build-something
  message: '{"ResourceType":"gcp-types/cloudbuild-v1:cloudbuild.projects.builds.create","ResourceErrorCode":"500","ResourceErrorMessage":"Build
    failed; check build logs for details"}'


Unfortunately the logs don't say much, except "FAILED_PRECONDITION". Did you run into this? Have I done something wrong?

Reply:
Did you give the Cloud Builder service account read access to DM?